jac-uk/apply
# 730: Reset Qt response and Qt related tests

## What's included?

This PR includes changes which support candidates ability to retake a QT if they have started and then been reset through the admin portal.
I have also included some unit tests for the tab functionality and created a working pattern to mock dates.
I have removed the unused `countdown` component and changed the `countdown2` to be the only QT timer, i have also added an initial test for the countdown.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
 To test you must have an open QT with access to a candidate account eligible to take the qualifying test.
 - Complete qualifying test. 
 - Observe the test shows in past tab, not in open and cannot be restarted.
 - Use accompanying #1226 admin ticket to reset candidate response.
 - Observe the test showing in open tests; can be started.
 - Resubmit test.

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

## Additional context
![image](https://user-images.githubusercontent.com/44227249/126511082-5d26b436-b4c3-4d54-9560-969d86137170.png)

Video: [here](https://drive.google.com/drive/u/1/folders/1-kkQ1_ArvpIiHIid7NBPDzwnPoss0TPR)

https://user-images.githubusercontent.com/44227249/126500643-c7ca230c-4f1b-4c95-8ba4-2ff4f13257d6.mov

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
